### ISSUE TITLE
could not determine interpretation of row comparison operator =

### ISSUE THREAD

**Issue by Huajianjin** [IvorySQL/IvorySQL#25]
## Bug Report
When multiple columns of two or more tables are of type varchar2 and are compared, could not determine interpretation of row comparison operator =.

### IvorySQL Version
1.0

### OS Version (uname -a)
Linux localhost.localdomain 4.18.0-80.el8.x86_64 #1 SMP Tue Jun 4 09:19:46 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

### Configuration options  ( config.status --config )
./configure --prefix=/home/IVORYSQL --enable-debug --enable-cassert  CFLAGS='-O0 -g' --enable-tap-tests

### Current Behavior
```
postgres=# select * from test_var where(c1,c2) not in (select * from test_tmp);
2021-12-21 15:02:21.648 CST [8344] ERROR:  could not determine interpretation of row comparison operator = at character 37
2021-12-21 15:02:21.648 CST [8344] HINT:  Row comparison operators must be associated with btree operator families.
2021-12-21 15:02:21.648 CST [8344] STATEMENT:  select * from test_var where(c1,c2) not in (select * from test_tmp);
ERROR:  could not determine interpretation of row comparison operator =
LINE 1: select * from test_var where(c1,c2) not in (select * from te...
                                            ^
HINT:  Row comparison operators must be associated with btree operator families.
```
### Expected behavior/code
```
postgres=# select * from test_var where(c1,c2) not in (select * from test_tmp);
 c1 |  c2   
----+-------
 2  | test2
(1 row)
```
### Step to reproduce
```
set search_path to oracle,pg_catalog;
create table test_var(c1 varchar2,c2 varchar2);
insert into test_var values('1','test1'),('1','test1'),('2','test1'),('2','test2');
create table test_tmp(c1 varchar2,c2 varchar2);
insert into test_tmp values('1','test1'),('2','test1');
select * from test_var where(c1,c2) not in (select * from test_tmp);
```
### Additional context that can be helpful for identifying the problem
Row comparison operators must be associated with btree operator families.

**Comment by BingAo-Office** [comment]
Resolved by #26

**Comment by rasifr** [comment]
The bug was fixed by [26](https://github.com/IvorySQL/IvorySQL/pull/26)
